test/integration/client_test.go

```diff
@@ -546,3 +546,258 @@ func TestClientFailoverReplication(t *testing.T) {
 	t.Log("Disable volume replication not ready!")
 }
```

Comment:
Should not this file be separated for different features? like client_fs_test, client_block_test , client_relication_test....?
@joseph-v @Shruthi-1MN Can we please consider this?